### PR TITLE
[#99] Fix Prettier tab indentation to Airbnb spec

### LINKED ISSUE #99: Codestyle enforcement currently not following Airbnb indentation rules
**Describe the bug**
Currently we are meant to be following the Airbnb codestyle guide, yet our indentation is 4 spaces rather than 2, so this should be rectified

**To Reproduce**
1. Make a change in any js,css,md or yaml file, and have a line with only 2 spaces for an indent
2. Save the file

**Expected behavior**
If prettier on save has been enabled, the line with a 2-space indent becomes a 4-space automatically

**Desktop (please complete the following information):**
 - OS: Win 11
 - IDE: VS Code

**Smartphone (please complete the following information):**
 - N/A
**Additional context**
N/A

### PR DESCRIPTION
**Pull request by harristomy**
# Description:
Minor fix, change tabWidth to 2 from 4

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](../CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project
- [ ] My code has been commented
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "trailingComma": "es5",
-    "tabWidth": 4,
+    "tabWidth": 2,
     "semi": false,
     "singleQuote": true
   }
```
Comment (seanhogunkim) [inline .prettierrc:6]: No newline at end of file might cause a linting error

Comment (harristomy) [reply .prettierrc:6]: You want me to add a prettier rule to always add a newline at EOF u mean?

Comment (seanhogunkim) [reply .prettierrc:6]: Oh, if there is no rule for that then it should be OK.

**Comment by seanhogunkim** [comment]
Tested on my end, everything seems to be working fine LGTM

**Review by seanhogunkim — APPROVED** [review]
Tested on my end, everything seems to be working fine LGTM

**Pull request merged**